README.md

```diff
@@ -1,44 +1,24 @@
-# TFLint Ruleset Template
-[![Build Status](https://github.com/terraform-linters/tflint-ruleset-template/workflows/build/badge.svg?branch=main)](https://github.com/terraform-linters/tflint-ruleset-template/actions)
+# TFLint kafka-config ruleset
 
-This is a template repository for building a custom ruleset. You can create a plugin repository from "Use this template". See also [Writing Plugins](https://github.com/terraform-linters/tflint/blob/master/docs/developer-guide/plugins.md).
-
-## Requirements
-
-- TFLint v0.42+
-- Go v1.22
+This is a [tflint](https://github.com/terraform-linters/tflint) [plugin](https://github.com/terraform-linters/tflint/blob/master/docs/developer-guide/plugins.md) for enforcing UW rules over our kafka config.
 
 ## Installation
 
-TODO: This template repository does not contain release binaries, so this installation will not work. Please rewrite for your repository. See the "Building the plugin" section to get this template ruleset working.
-
 You can install the plugin with `tflint --init`. Declare a config in `.tflint.hcl` as follows:
 
 ```hcl
-plugin "template" {
+plugin "kafka-config" {
   enabled = true
 
-  version = "0.1.0"
-  source  = "github.com/terraform-linters/tflint-ruleset-template"
-
-  signing_key = <<-KEY
-  -----BEGIN PGP PUBLIC KEY BLOCK-----
-  mQINBGCqS2YBEADJ7gHktSV5NgUe08hD/uWWPwY07d5WZ1+F9I9SoiK/mtcNGz4P
-  JLrYAIUTMBvrxk3I+kuwhp7MCk7CD/tRVkPRIklONgtKsp8jCke7FB3PuFlP/ptL
-  SlbaXx53FCZSOzCJo9puZajVWydoGfnZi5apddd11Zw1FuJma3YElHZ1A1D2YvrF
-  ...
-  KEY
+  version = "x.y.z"
+  source  = "github.com/utilitywarehouse/tflint-ruleset-kafka-config"
```

Comment:
I'm wondering: if this repo were private would this still work as long as `git` was configured with some credentials to clone it? But I have no objections to having it public

Reply:
tflint doesn't seem to support fetching from private repos

Reply:
> tflint doesn't seem to support fetching from private repos, as it needs to download the binary from the release. It doesn't clone

Ah, it looks to do asset fetching via the GitHub API (rather than cloning the repo and building) https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/plugins.md#avoiding-rate-limiting so I think passing it a `GITHUB_TOKEN` env var with permissions to the private repo _should_ work, but again, don't really need to